src/lib/variables/variables.test.ts

```diff
@@ -10,6 +11,28 @@ import {
 } from './variables';
 
 describe('Variables', () => {
+  describe('isClassInstance', () => {
+    class C {
+      // empty
+    }
+
+    it.each([new C()])('returns true for %p', input => {
```

Comment:
This pretty print isn't ideal, I will need to fix it, so test results are readable.